<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1474</width>
    <height>1115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="一款让你心动的笔记办公软件">个人笔记</string>
  </property>
  <property name="windowIcon">
   <iconset resource="file.qrc">
    <normaloff>:/app_icon.png</normaloff>:/app_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Arial&quot;;</string>
        </property>
       </widget>
       <widget class="QAxWidget" name="axWidget"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1474</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_create"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_saveAs"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="edit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_C"/>
    <addaction name="action_V"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="window">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
    <addaction name="window_max"/>
    <addaction name="window_min"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>设置
</string>
     </property>
     <addaction name="action_3"/>
     <addaction name="action_4"/>
     <addaction name="action_5"/>
    </widget>
    <addaction name="action_about"/>
    <addaction name="menu"/>
   </widget>
   <addaction name="file"/>
   <addaction name="edit"/>
   <addaction name="window"/>
   <addaction name="help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_create"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="action_saveAs"/>
   <addaction name="separator"/>
   <addaction name="action_C"/>
   <addaction name="action_V"/>
   <addaction name="action_X"/>
   <addaction name="action_3"/>
   <addaction name="action_4"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_create">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/newfile.png</normaloff>:/icon/newfile.png</iconset>
   </property>
   <property name="text">
    <string>新建文件(&amp;N)</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/open.png</normaloff>:/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/save.png</normaloff>:/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/saveAs.png</normaloff>:/icon/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/exit.png</normaloff>:/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/C.png</normaloff>:/icon/C.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="action_V">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/V.png</normaloff>:/icon/V.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
  </action>
  <action name="action_X">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/X.png</normaloff>:/icon/X.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;X)</string>
   </property>
  </action>
  <action name="window_max">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/max_window.png</normaloff>:/icon/max_window.png</iconset>
   </property>
   <property name="text">
    <string>最大化</string>
   </property>
  </action>
  <action name="window_min">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/min_window.png</normaloff>:/icon/min_window.png</iconset>
   </property>
   <property name="text">
    <string>最小化</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/about.png</normaloff>:/icon/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/setting.png</normaloff>:/icon/setting.png</iconset>
   </property>
   <property name="text">
    <string>设置
</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/fontsize.png</normaloff>:/icon/fontsize.png</iconset>
   </property>
   <property name="text">
    <string>字体大小</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="file.qrc">
     <normaloff>:/icon/fontcolor.png</normaloff>:/icon/fontcolor.png</iconset>
   </property>
   <property name="text">
    <string>字体颜色</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>保存路径</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="file.qrc"/>
 </resources>
 <connections/>
</ui>
